src/app/pages/BibPage.jsx

```diff
@@ -90,11 +91,8 @@ export const BibPage = (
   // const bNumber = bib && bib.idBnum ? bib.idBnum : '';
   const itemPage = location.search;
   const aggregatedElectronicResources = getAggregatedElectronicResources(items);
-  let shortenItems = true;
 
```

Comment:
Forgot I did this.

The purpose here was to shorten the logic required to check if the items needed to be shortened. 
Simply, if the indexOf('all') is 1: true Else it's -1: false

Reply:
It's the first commit in this PR